.github/workflows/zoningtaxlots_build.yml

```diff
@@ -23,30 +29,26 @@ jobs:
           --health-timeout 5s
           --health-retries 5
         ports:
-          - 5435:5432
+          - 5432:5432
     env:
-      BUILD_ENGINE: postgresql://postgres:postgres@localhost:5435/postgres
+      BUILD_ENGINE: postgresql://postgres:postgres@postgis:5432/postgres
       EDM_DATA: ${{ secrets.EDM_DATA }}
       AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
       AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
     steps:
       - uses: actions/checkout@v3
-      - name: install dependencies ...
-        run: |
-          sudo apt update
-          sudo apt install -y gdal-bin
-
-          curl -O https://dl.min.io/client/mc/release/linux-amd64/mc
-          chmod +x mc
-          sudo mv ./mc /usr/bin
-          mc config host add spaces $AWS_S3_ENDPOINT $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY --api S3v4
 
+      - name: Finish container setup
+        working-directory: ./
+        run: ./bash/docker_container_setup.sh
+        
       - name: 1. dataloading ...
         run: ./ztl.sh dataloading
 
       - name: 2. build ...
         run: ./ztl.sh build
 
-      - name: 3. qaqc ...
+      - name: 3. qaqc, export, and upload ...
```

Comment:
Would have split these into multiple bash scripts but for ztl they're kind of intertwined